config/ember-try.js

```diff
@@ -12,24 +12,42 @@ module.exports = function() {
       scenarios: [
         {
           name: 'ember-lts-2.12',
+          env: {
+            EMBER_OPTIONAL_FEATURES: JSON.stringify({
+              'jquery-integration': true
```

Comment:
This is part of the default `ember-try` config now, but we probably don't actually care about testing with and without jQuery since there's no dependency on it anywhere in this addon.

Reply:
The way it's implemented now means that all scenarios are executed `jquery-integration` turned on, right? So I guess we should remove all `@ember/jquery` deps in here and set `jquery-integration` to `false` as well.

Reply:
Yes; that would probably make sense for this addon. I can do that in this PR or in a follow-up.

Reply:
I don't mind either way. 😊

Reply:
I'll do it in a separate PR